website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -9,30 +10,37 @@ id: "metrics"
 
 </Changelog>
 
+
 :::info Metrics are new
-An initial version of `metrics` was introduced in v1.0.0, following [vibrant community discusion](https://github.com/dbt-labs/dbt-core/issues/4071). Try them out, and let us know what you think!
+v1.0.0 includes an initial version of metrics, following a [vibrant community discussion](https://github.com/dbt-labs/dbt-core/issues/4071). Try them out, and let us know what you think!
 :::
 
 :::caution Metrics are experimental
-`metrics` will be released in v1.0, but they should _not_ be considered a stable API. We reserve the right to make breaking changes to their schema in future **minor** versions. We will aim for backwards compatibility whenever possible.
+v1.0 includes metrics, but they should be considered an _unstable_ API because they are experimental and subject to change. We reserve the right to make breaking changes to the metrics schema in future **minor** versions, but will aim for backwards compatibility when possible.
 :::
 
-## Related documentation
-* [`metric:` selection method](node-selection/methods#the-metric-method)
-
-## Getting started
+## About metrics 
 
 A metric is a timeseries aggregation over a table that supports zero or more dimensions. Some examples of metrics include:
-
 - active users
 - churn rate
 - mrr (monthly recurring revenue)
 
-Starting in v1.0, dbt supports metric definitions as a new node type. Like [exposures](exposures), metrics participate in the dbt DAG and can be expressed in yaml files. By defining metrics in dbt projects, analytics engineers can encode crucial business logic in tested, version controlled code. Further, these metrics definitions can be exposed to downstream tooling to drive consistency and precision in metric reporting.
+In v1.0, dbt supports metric definitions as a new node type. Like [exposures](exposures), metrics participate in the dbt DAG and can be expressed in YAML files. By defining metrics in dbt projects, you can encode crucial business logic in tested, version-controlled code. Further, you can expose these metrics definitions to downstream tooling, which drives consistency and precision in metric reporting.
+
+### Benefits of defining metrics
```

Comment:
I moved this up and changed its title because it feels like part of the description.